uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -829,7 +856,13 @@ define([
       function updateRequest() {
         vm.leaveRequest.update()
           .then(function () {
-            postSubmit('LeaveRequest::updatedByManager');
+            if (vm.role == 'manager') {
+              postSubmit('LeaveRequest::updatedByManager');
```

Comment:
the action maybe edit but the recipient are different, hence separate event, also manager could be owner for its leaverequest and that edit will be different from manager role edit